frontend/src/styles/AppStyled.js

```diff
@@ -0,0 +1,11 @@
+import { createGlobalStyle } from "styled-components";
+
+const AppStyled = createGlobalStyle`
+body,
+html {
+  font-family: "Open Sans", sans-serif;
```

Comment:
Ezt akartam is kérni, de ha már így van, hogy ez a default font, akkor a komponenseken belül nem indokoltak ugyanezek a sorok, nem?